arch/arm/boot/Makefile

```diff
@@ -32,7 +35,27 @@ INITRD_PHYS := $(initrd_phys-y)
 
 export ZRELADDR INITRD_PHYS PARAMS_PHYS
 
-targets := Image zImage xipImage bootpImage uImage
+targets := Image zImage xipImage bootpImage uImage dt-blob
+
+$(obj)/dt-blob: FORCE
```

Comment:
Remove that dt-blob crap!!!!! Do it cleanly!!!

Bloating an already bloated QC kernel with bloaty bloat is... bloaty? :)